legal-api/src/legal_api/core/filing.py

```diff
@@ -117,15 +160,19 @@ def storage(self, filing: FilingStorage):
         """Set filing storage."""
         self._storage = filing
 
-    @json.setter
-    def json(self, filing_submission):
-        """Add the raw json to the filing."""
-        self._raw = filing_submission
-
     def save(self):
         """Save the filing."""
-        self.storage.filing_json = self._raw
-        self._storage.save()
+        raise NotImplementedError
+
+    def diff(self):
+        """Return the diff block for the filing this one corrects, if any."""
+        if self.json and self.status == Filing.Status.COMPLETED.value:
+            if correction_id := self.json.get('correction'):
```

Comment:
Should this be looking for correctedFilingId inside correction object

Reply:
Yep, will get that in with the test to cover it.